site/welcome.rst

```diff
@@ -6,60 +6,112 @@
+               See how to get started with deploying your software using Nuitka, it's
+               easy and we are there to guide you and help you through it. Many people
+               say it works out of the box, but if your software is very complex, we
+               can help you get it done.
 
-      Dear IT Manager,
+            .. carousel-cta::
+               :url: https://example.com
 
-      see how your business can benefit from Nuitka with IP protection and its
-      powerful deployment solutions. Also, see the many convenience features
-      that can save your team a bunch of development time.
+               Take the Deployment Tour <i class="fa fa-arrow-right sd-text-white"></i>
 
-      Take the IT manager tour
+         .. carousel-main-content::
 
-   .. tab-item:: Advanced Pythonista
-      :class-label: carousel-label carousel-persona-laura
-      :class-container: carousel-container carousel-persona-laura
-      :class-content: carousel-content carousel-persona-laura
+            .. carousel-main-content-heading::
 
-      Dear Python expert,
+               Dear Python Beginner,
 
-      you have ideas about what Nuitka is, and we are here to answer them in
-      great detail (eventually). Explore the power of Nuitka for deployment,
-      performance, and IP protection with more details and expert knowledge
-      used.
+            .. carousel-main-content-text::
 
-      Take the Advanced Tour
+               You and I know you are not at all familiar and you need help. Get
+               potentially instant help from a community of volunteers.
+
+            .. carousel-cta::
+               :url: https://example.com
+
+               Take the Beginner Tour <i class="fa fa-arrow-right sd-text-white"></i>
+
+         .. carousel-main-content::
+
+            .. carousel-main-content-heading::
+
+               Dear IT Manager,
+
+            .. carousel-main-content-text::
+
+               See how your business can benefit from Nuitka with IP protection and its
+               powerful deployment solutions. Also, see the many convenience features
+               that can save your team a bunch of development time.
+
+            .. carousel-cta::
+               :url: https://example.com
+
+               Take the IT Manager Tour <i class="fa fa-arrow-right sd-text-white"></i>
+
+         .. carousel-main-content::
+
+            .. carousel-main-content-heading::
+
+               Dear Python Expert,
+
+            .. carousel-main-content-text::
+
+               You have ideas about what Nuitka is, and we are here to answer them in
+               great detail (eventually). Explore the power of Nuitka for deployment,
+               performance, and IP protection with more details and expert knowledge
+               used.
+
+            .. carousel-cta::
+               :url: https://example.com
+
+               Take the Advanced Tour <i class="fa fa-arrow-right sd-text-white"></i>
```

Comment:
Use replacements like `|CREDITS_SYMBOL|` and it would appear, maybe instead of an URL attribute.

They are defined in variables.inc like so:

```
.. |CREDITS_SYMBOL| raw:: html

   <i class="fa fa-fw fa-users" aria-hidden="true"></i>
```

That is the standard way of including raw HTML and Font Awesome icons specifically. As for the color, I believe, a CSS rule or post processing ought to add those, don't make them part of the HTML in the document here.